include/magma/vk/buffers/BufferManager.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file BifferManager.h
+ * @brief Tool for buffers creation and control
+ * @version 0.1
+ * @date 2021-02-18
+ */
+#pragma once
+#include <string>
+#include <unordered_map>
+
+#include "magma/app/log.hpp"
+#include "magma/vk/commandBuffer.h"
+#include "magma/vk/buffers/Buffer.h"
+
+class LogicalDevice;
+
+class BufferManager {
+public:
+    BufferManager(LogicalDevice &device);
+    ~BufferManager();
+
+    bool bufferExists(const std::string &name) const;
+    Buffer& getBuffer(const std::string &name);
+
+    Buffer& createBuffer(const std::string &name, VkDeviceSize size,
+        VkBufferUsageFlags usage, VkMemoryPropertyFlags properties);
+    Buffer& createHostBuffer(const std::string &name, VkDeviceSize size, VkBufferUsageFlags usage);
```

Comment:
I don't like this function.
It creates not just HostBuffer but HOST_VISIBLE | HOST_COHERENT what is not obvious actually